packages/common/src/index.ts

```diff
@@ -494,7 +494,7 @@ export type ApiCompiledQueryResults = string;
 
 export type ApiExploresResults = SummaryExplore[];
 
-export type ApiExploreResults = Explore;
+export type ApiExploreResults = Omit<Explore, 'unfilteredTables'>;
```

Comment:
Have you run `generate-api` to check there aren't other affected response types?

Reply:
on swagger, all affected endpoints replaced `Explore` with `Pick_Explore.Exclude_keyofExplore.unfilteredTables__`